src/ExtInterface/ReefMod/RMEDomain.jl

```diff
@@ -140,34 +140,19 @@ function load_domain(
     isempty(single_reef_idx) && push!(single_reef_idx, 1)
     force_single_reef && (spatial_data = spatial_data[[1], :])
 
-    # Find start year
+    # Find initial coral cover start year
     initial_csv_files = readdir(joinpath(data_files, "initial_csv"))
     icc_filename = initial_csv_files[occursin.("coral_sp", initial_csv_files)][1]
+    icc_start_year::Int64 = parse(Int64, split(icc_filename, r"[_.]")[end - 1])
 
-    # Load DHW scens
-    dhw_scens::YAXArray{Float64} = load_DHW(RMEDomain, data_files, RCP)
+    # Load dhw_scens
```

Comment:
Small disagreement with the change here.

If the intent is to improve clarity, then might as well expand it rather than replace with a variable name (the variable name doesn't add more context in my view).

```suggestion
    # Load DHW scenarios
```